view-and-query-asset-inventory.md

```diff
@@ -40,6 +40,17 @@ resources
 | where ['type'] contains "microsoft.hybridcompute" or ['type'] contains "microsoft.compute"
 ```
 
+##### Scenario: query for all virtual machines in Azure and AWS along with their instance size
+```
+resources 
+| where subscriptionId == "<yoursubscriptionid>"
+| where (['type'] == "microsoft.compute/virtualmachines") 
+| union (awsresources | where (type == "microsoft.awsconnector/ec2instances" and subscriptionId == "3ced23ce-e34c-4455-a687-9f183e1ac302"))
```

Comment:
replace 3ced23ce-e34c-4455-a687-9f183e1ac302 with "yourscriptionid"

We can remove the subscriptionId so the query will show all virtual machines in the whole tenant?